CalibTracker/SiStripESProducers/plugins/real/SiStripLorentzAngleDepESProducer.cc

```diff
@@ -36,7 +36,7 @@ SiStripLorentzAngleDepESProducer::SiStripLorentzAngleDepESProducer(const edm::Pa
 
 std::shared_ptr<SiStripLorentzAngle> SiStripLorentzAngleDepESProducer::produce(const SiStripLorentzAngleDepRcd& iRecord)
 {
-
+  std::shared_ptr<SiStripLorentzAngle> siStripLA_;
```

Comment:
Please remove the underscore in the name, since CMS convention uses a trailing underscore as a way to denote a member data.